terminal_colors/terminal_colors.py

```diff
@@ -111,13 +120,10 @@ def _get_options(args):
     parser.add_option_group(group)
 
     (opts, args) = parser.parse_args(args)
-    return opts
+    # OptionParser does not allow combining choices= with type="int", so...
+    opts.force_term = None if opts.force_term is None else int(opts.force_term)
 
-# instantiate global options based on command arguments
-options = _get_options(sys.argv[1:])
-# don't allow -f by itself
-options.foreground = options.foreground and (
-        options.numbers or options.hex or options.rgb)
+    return opts
 
 class _staticmethods(type):
```

Comment:
fyi VSCod[e/ium] with Pylance doesn't understand this metaclass trick and reports error-level problems about the first arg to the class functions. It might be worth just typing in the @staticmethod on the static methods :)